src/FSharp.Markdown/HtmlFormatting.fs

```diff
@@ -182,11 +182,18 @@ let rec formatParagraph (ctx:FormattingContext) paragraph =
       ctx.Writer.Write("</p>")
   | HorizontalRule(_) ->
       ctx.Writer.Write("<hr />")
+  | CodeBlock(String.StartsWithWrapped ("[", "]") (ParseCommands cmds, String.TrimStart code)) 
```

Comment:
I'm not completely sure if I understand the issue here. I figured that the html output with the `language-*` css classes would be more useful. Why do you consider this code as language specific?

Besides that I think it is no problem removing this part of the PR (the FSharp.Literate stuff doesn't depend on this) if you don't like it.